packages/components/Callout/src/Callout.types.ts

```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
+import * as ReactNative from 'react-native';
+import { IRenderData } from '@uifabricshared/foundation-composable';
+import { IBackgroundColorTokens, IBorderTokens } from '@fluentui-react-native/tokens';
+import { IFocusable } from '@fluentui-react-native/interactive-hooks';
+
+export const calloutName = 'RNFCallout';
+
+/**
+ * Properties and Tokens for fabric native Callout
+ */
+
+export type DirectionalHint =
+  | 'leftTopEdge'
+  | 'leftCenter'
+  | 'leftBottomEdge'
+  | 'topLeftEdge'
+  | 'topAutoEdge'
+  | 'topCenter'
+  | 'topRightEdge'
+  | 'rightTopEdge'
+  | 'rightCenter'
+  | 'rightBottomEdge'
+  | 'bottonLeftEdge'
+  | 'bottomAutoEdge'
+  | 'bottomCenter'
+  | 'bottomRightEdge';
+
+export interface ICalloutTokens extends IBackgroundColorTokens, IBorderTokens {
+  anchorRect?: ReactNative.ScreenRect;
+  directionalHint?: DirectionalHint;
+  gapSpace?: number;
+  minPadding?: number;
+}
+
```

Comment:
It's a little curious to me that your using Tokens & Composable.  Are you adding a compose version later?